_layouts/default-grid.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
The only difference is one has a grid around the content and the other does not. We could remove the grid one and add it manually for each page. But the advantage of having grid in the template is the page content can be pure markdown and easier for people to update and add new pages.